### ISSUE TITLE
A11y_Azure Data Studio_Azure Menu _Screenreader: Screenreader does not announcing "Presse Shift+F10 for expand Azure Menu" when keyboard focus is on Azure Tree Item.

### ISSUE THREAD

**Issue by anthonydresser** [anthonydresser/testissues#4731]
“Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: -A11yMAS;-A11yTCS;-A11ySev2;-SH_AzureDataStudio_Feb2021;-SQL Azure Data Studio;-Win10;-FTP;-DesktopApp;-WCAG3.3.2;-NonBenchmark;-Screenreader;-JAWS;-NVDA;-STP;

Environment Details:
Application Name: Azure Data Studio
Version: 1.26.0-insider (user setup)
Windows Version: Windows10
Screenreader: JAWS, NVDA

**Repro Steps:**
1.Launch Azure Data Studio insiders application.
2.Navigate to Connections from left menu and in servers button and hit enter "New Connection" button.
3.Connections section will get open.
4.Tab to "Browse Preview" Tab item and hit enter.
5.Navigate to "Azure" tree item.
6.Start JAWS and check jaws is announcing shortcut key Shift+F10 for expand Azure Menu.

**Actual:**
Screenreader does not announcing "Presse Shift+F10 for expand Azure Menu" when keyboard focus is on Azure Tree Item.

**Expected:**
When keyboard focus is on Azure Tree Item screenreader Should also announce shortcut key detail that "Press Shift+F10 for expand Azure Menu".

**User Impact:**
Screenreader users will not get information regarding how to expand Azure Menu if screenreader does not announce press Shift+F10 for expand Azure menu.
